mdbenchmark/mdengines/namd.py

```diff
@@ -28,6 +28,9 @@
 def prepare_benchmark(name, relative_path, *args, **kwargs):
     benchmark = kwargs["benchmark"]
 
+    if not kwargs["multidir"] == 1:
+        raise console.error("The NAMD-engine currently only supports '--multidir 1'")
```

Comment:
We do not need to raise `console.error`. Calling it will ultimately call `sys.exit()` and exit gracefully.

```suggestion
        console.error("The NAMD-engine currently only supports '--multidir 1'")
```

Reply:
uups! sorry I knew that actually.